lib/host.py

```diff
@@ -406,6 +406,9 @@ def available_disks(self):
         """
         return [disk for disk in self.disks() if self.disk_is_available(disk)]
 
+    def disk_devid(self, disk):
+        return self.ssh(["find", "-L", "/dev/disk/by-id/", "-samefile", "/dev/" + disk]).splitlines()
```

Comment:
Will the dev-id be the same on all hosts?

Reply:
I'm not sure I understand. A disk devid is linked to the host we get it from like the disk is originally.

Reply:
sr_disk will respond the same answer on all hosts (for instance `/sdb`)
if the id returned changes on each hosts this fixture wont work on pools with several hosts.

Reply:
Actually `sr_disk` is only a fixture for the master host, contrarily to `sr_disk_for_all_hosts`. So I see no problem with a fixture which just extracts this ID from the disk, as long as we don't try to reuse the dev-id on other hosts of the pool.